graphistry/text_utils.py

```diff
@@ -272,10 +272,16 @@ def search_graph(
 
     def save_search_instance(self, savepath):
         from joblib import dump  # type: ignore   # need to make this onnx or similar
-
-        self.build_index()
-        search = self.search_index
-        del self.search_index  # can't pickle Annoy
+        try:
+            self.build_index()
+            search = self.search_index
+            del self.search_index  # can't pickle Annoy
+        except Exception as e:
+            logger.exception(e)
+            logger.warn(
```

Comment:
should this be `import warnings; ...` ? i'm not that versed on python, but to enable structured unit testing, safer sw, etc